transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollChannelOption.java

```diff
@@ -18,6 +18,10 @@
 import io.netty.channel.ChannelOption;
 
 public final class EpollChannelOption<T> extends ChannelOption<T> {
+    public enum EpollDomainSocketReadMode {
```

Comment:
I think we should just keep it in epoll package for now as we not have any other native implementations anyway. If we really add some later we may need to move some stuff anyway for better reuse.